docs/multicluster/user-guide.md

```diff
@@ -805,6 +805,61 @@ the image.
 3. Copy the image file `antrea-mcs.tar` to the Nodes of your local cluster.
 4. Run `docker load < antrea-mcs.tar` in each Node of your local cluster.
 
+## Remove a member cluster
+
+When you want to remove a member cluster in a ClusterSet, please follow up
+the following steps to clean up all Multi-cluster related resources.
+
+Note: Please replace `kube-system` with the right one in the following sample
+commands and manifest if the Antrea Multi-cluster is not deployed in the
+default Namespace.
+
+1. Delete the existing ClusterSet CR. The Antrea Multi-cluster Controller will
+be responsible to clean up all resources created by itself automatically:
+
+```bash
+kubectl get clusterset -n kube-system -o=jsonpath='{.items[0].metadata.name}' | xargs -I{} kubectl delete clusterset {} -n kube-system
+```
+
+2. Delete the Antrea Multi-cluster deployment:
+
+```bash
+kubectl delete -f https://github.com/antrea-io/antrea/releases/download/$TAG/antrea-multicluster-member.yml
+```
+
+## Remove a Leader Cluster
+
+When you want to remove the leader cluster in a ClusterSet, please follow up
+the following steps to clean up all Multi-cluster related resources. Please notes
+that you need to follow the section [Remove a member cluster](#remove-a-member-cluster)
+to remove all member clusters before removing a leader cluster.
+
+Note: Please replace `antrea-multicluster` with the right one in the following sample
+commands and manifest if the Antrea Multi-cluster is not deployed in the default Namespace.
+
+1. Verify that there is no remaining MemberClusterAnnounces: `kubectl get memberclusterannounce -n antrea-multicluster`.
```

Comment:
Antrea MC controller will also clean up this kind of ResoureExports automatically when the ClusterSet CR is deleted, do you mean to add a step to let user to delete them manually since it's created by the admin or user?

Reply:
ANPExport is created by users in the leader? We should not auto-delete user created CRs.

Reply:
ACNP kind of ResourceExports will be created by the admin for ACNP replication.
For other ClusterSet level NP, it's defined by users in each member clusters. I feel we can ignore this part?

Reply:
Skip deleting ACNP type of ResourceExports and add a step to allow users to delete them manually.